### PR TITLE
Include B1+ fieldmaps in main specification

### PR DESCRIPTION
**Pull request by Gilles86**
Following up on issue #44.

Would especially like some feedback from co-assignee @agahkarakuzu.

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -650,3 +663,32 @@ file to obtain scanning parameters. \_epi files can be a 3D or 4D - in the
 latter case all timepoints share the same scanning parameters. To indicate which
 run is intended to be used with which functional or diffusion scan the
 IntendedFor field in the JSON file should be used.
+
+#### B1+ fieldmaps
```
Comment (agahkarakuzu) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:667]: @Gilles86 these additions  look great! Should we go ahead and define some B1 mapping methods (e.g. double angle for B1+) in the suffix table, or would you rather prefer to do it later?

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -496,10 +496,23 @@ JSON example:
...
-acquired for by filling the IntendedFor field in the corresponding JSON file.
+Both B0 (static magnetic field strength pattern), B1+ (transmit field pattern), and 
+B1- (receive field pattern; not yet supported) maps can be useful in post-processing
+both raw functional and anatomical data.
+
+B0 maps are primarily used to correct for spatial distortions in functional
+data acquired with EPI sequences.
+
+B1+ and B1- maps are mostly used in anatomical imaging, especially when applying
+quantitative MRI (qMRI or hMRI) techniques.
```
Comment (agahkarakuzu) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: Minor concern, `quantitative MRI (qMRI or hMRI)` still confuses me :)

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md (hunk 1 above)
Comment (Gilles86) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:667]: Sure. I have very little expertise about this though!

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md (hunk 2 above)
Comment (Gilles86) [reply src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: Yeah good point. Let's just keep it `qmri`.

**Review by KirstieJane — APPROVED** [review]
🚀 ✨ 

Reviewed in the May meeting. ready to merge!

**Pull request merged**